ansible/files/bootstrap/telemetry.sh

```diff
@@ -137,6 +145,28 @@ sudo yum makecache
 # Install packages
 sudo yum -y install git jq htop tmux libffi-devel aws-cli postgresql-devel zsh snappy-devel readline-devel emacs nethogs w3m
 
+# Install agents for metrics and logging
+if [ "$METRICS_PROVIDER" = "stackdriver" ] || [ "$LOGGING_PROVIDER" = "stackdriver" ]; then
+    # Install GCP credentials for stackdriver; see https://cloud.google.com/logging/docs/agent/authorization
+    sudo mkdir -p /etc/google/auth/
+    sudo aws s3 cp $TELEMETRY_CONF_BUCKET/credentials/gcp.json /etc/google/auth/application_default_credentials.json
```

Comment:
who will have access to this bucket?

Reply:
Just checked and this bucket lives in our cloudservices-aws-dev account, so all developers have access to it.  A better practice would be to generate credentials(which we would have to do for the aws link gcp project anyway) and put them in a s3 bucket which only the EMR iam has access to.  This is annoying because we either have to host that s3 bucket in a separate aws account and set up cross account access, or add a rule to explicitly deny access to this bucket to the group that all developers belong to